### ISSUE TITLE
Clean up / add utils from od

### ISSUE THREAD

**Issue by benstoltz** [Esri/cedar#261]
Comments from @tomwayson:

Also need to:
Move tests from -ui to addon
Remove these lines:
https://github.com/ArcGIS/opendata-ui/blob/2bac928012f0c3a2857beee84c610457f02b6b20/packages/opendata-ui/ember-cli-build.js#L146
https://github.com/ArcGIS/opendata-ui/blob/2bac928012f0c3a2857beee84c610457f02b6b20/packages/opendata-admin/ember-cli-build.js#L125
They should not be needed after this:
https://github.com/ArcGIS/opendata-ui/blob/2bac928012f0c3a2857beee84c610457f02b6b20/packages/ember-cli-opendata-itempicker/index.js#L10

and

After discussing I learned that the utils are in the -cards addon too!
probably want to move isChartableField() logic into own library, then remove chart-utils from picker addon along w/ this line https://github.com/ArcGIS/opendata-ui/blob/2bac928012f0c3a2857beee84c610457f02b6b20/packages/ember-cli-opendata-itempicker/index.js#L10
Then, need to:
Add that line to the -cards addon index and add bower install of moment into it’s blueprint here:
https://github.com/ArcGIS/opendata-ui/blob/master/packages/ember-arcgis-layout-cards/blueprints/ember-cli-opendata-pages/index.js
Then update -ui to get any chartUtil code from -cards instead
Sigh

TLDR we need to grab some logic from od-ui and admin and move them into either cedar proper or a standalone utils lib for handling duplication of code.
